go-controller/pkg/cni/helper_linux.go

```diff
@@ -324,6 +324,14 @@ func ConfigureOVS(ctx context.Context, namespace, podName, hostIfaceName string,
 
 	ifaceID := util.GetIfaceId(namespace, podName)
 	initialPodUID := ifInfo.PodUID
+	pod, err := getPod(podLister, kclient, namespace, podName)
+	if IsStaticPod(pod) {
+		if err == nil {
+			initialPodUID = string(pod.ObjectMeta.UID)
+		} else {
+			klog.Infof("Could not find pod %v %v", pod, err)
```

Comment:
not Info... maybe a Warning?
Interestingly, here we are not paying attention if getPod failed when checking IsStaticPod. Maybe do the same at the other place where this is being called? We should be consistent about how/when we handle failures in getPod, agree?

Reply:
Agree, changed to warning and followed same procedure on the other function call